package.json

```diff
@@ -19,7 +19,7 @@
     "start:cervical-cancer": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources packages/esm-cervical-cancer-app",
     "start:pmtct": "openmrs develop --backend https://ohri-namibia-dev.globalhealthapp.net --sources packages/esm-ohri-pmtct-app",
     "start:form-render": "openmrs develop --backend https://ohri-demo.globalhealthapp.net --sources packages/esm-form-render-app",
-    "start:namibia": "openmrs develop --backend https://ohri-namibia.globalhealthapp.net/ --sources 'packages/esm-*-app/'",
+    "start:namibia": "openmrs develop --backend https://ohri-namibia-dev.globalhealthapp.net --sources 'packages/esm-*-app/'",
```

Comment:
@pirupius I think we had to change this to the new url for Namibia other than that we have a 502